openapi.asyncapi.proxy/docker/compose/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+services:
+  zilla:
+    image: ghcr.io/aklivity/zilla:${ZILLA_VERSION}
+    container_name: zilla
+    pull_policy: always
+    restart: unless-stopped
+    ports:
+      - 7114:7114
+      - 4444:4444
+    environment:
+      ZILLA_INCUBATOR_ENABLED: "true"
+      #JAVA_OPTIONS: -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:4444
+    volumes:
+      - ../../specs/http-openapi.yaml:/etc/zilla/http-openapi.yaml
+      - ../../specs/kafka-asyncapi.yaml:/etc/zilla/kafka-asyncapi.yaml
+      - ../../zilla.yaml:/etc/zilla/zilla.yaml
+    command: start -v -e -Pzilla.engine.verbose.composites=true
+
```

Comment:
Does a Kafka service need to be added to the compose stack?

Reply:
Not yet. 😉